app/components/primer/alpha/tool-tip-element.ts

```diff
@@ -237,6 +234,18 @@ class TooltipElement extends HTMLElement {
 
   static observedAttributes = ['data-type', 'data-direction', 'id', 'hidden']
 
+  #update() {
+    if (this.hidden) {
+      this.classList.remove(TOOLTIP_OPEN_CLASS, ...DIRECTION_CLASSES)
+    } else {
+      this.classList.add(TOOLTIP_OPEN_CLASS)
+      for (const tooltip of this.ownerDocument.querySelectorAll<HTMLElement>(this.tagName)) {
+        if (tooltip !== this) tooltip.hidden = true
+      }
+      this.#updatePosition()
+    }
+  }
+
```

Comment:
This fixes: [no-dom-traversal-in-attributechangedcallback](https://github.com/github/eslint-plugin-custom-elements/blob/main/docs/rules/no-dom-traversal-in-attributechangedcallback.md)